libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

```diff
@@ -295,8 +298,8 @@ export async function getFolderItems(
     const connectionId = connectionReference?.connection.id as string;
     const { operationId, parameters: referenceParameters } = selectedValue ? browse : open;
     const pickerParameters = Object.keys(referenceParameters ?? {}).reduce((result: Record<string, any>, paramKey: string) => {
-      if (referenceParameters?.[paramKey]?.selectedItemValuePath) {
-        return { ...result, [paramKey]: getPropertyValue(selectedValue, referenceParameters[paramKey].selectedItemValuePath) };
+      if (referenceParameters?.[paramKey]?.['value-property']) {
+        return { ...result, [paramKey]: getPropertyValue(selectedValue, referenceParameters[paramKey]['value-property']) };
```

Comment:
The reference parameter can have either of these properties, so please check first for selectedItemValuePath and then for value-property
for backcompatibility